### ISSUE TITLE
TTC error: received code 28 during stmt reading

### ISSUE THREAD

**Issue by angYuHao** [sijms/go-ora#255]
This problem occurs  when I execute ‘alter database open read only’ 
I repeated it many times the result is 'TTC error: received code 28 during stmt reading'

**Comment by sijms** [comment]
can you give your example 

this is the code I use
```golang
package main

import (
	"database/sql"
	"fmt"
	go_ora "github.com/sijms/go-ora/v2"
)

func main() {
	connStr := go_ora.BuildUrl("localhost", 1521, "lab", "user", "pass", nil)
	conn, err := sql.Open("oracle", connStr)
	if err != nil {
		fmt.Println("connection error: ", err)
		return
	}
	defer func() {
		err = conn.Close()
		if err != nil {
			fmt.Println("error during conn close: ", err)
		}
	}()
	_, err = conn.Exec(`alter database open read only`)
	if err != nil {
		fmt.Println("execute error: ", err)
		return
	}
}
```

and this is the result
```
execute error:  ORA-65019: pluggable database LAB already open
```

**Comment by angYuHao** [comment]
my oracle version is 19c  when I set it to the mount status  I execute the same code as yours
although the program returns a failure, the oracle database does change from mount status to read only status
![image](https://user-images.githubusercontent.com/37206854/202837472-21d7d275-b82d-4f63-bcf7-0af2af86b53e.png)
If I ignore the error returned by execution, the execution of the second command will also return the TTC error

**Comment by sijms** [comment]
i do the following
```sql
shutdown immediate
stratup mount
```

then run the code above
the result is as follow
```
execute error:  ORA-01033: ORACLE initialization or shutdown in progress
```

**Comment by sijms** [comment]
I think this command for shutdown and startup need connection using OS authentication not network authentication
because i run same command from OS authentication inside server and command run successfully

**Comment by sijms** [comment]
I find the bug related to oracle 11g Summary Object reading and fix it in version 2.5.11
pls test it and see if every thing is ok
